torch/_inductor/utils.py

```diff
@@ -42,6 +45,8 @@
 from typing_extensions import Concatenate, ParamSpec
 
 import torch
+import torch._export
```

Comment:
This import is really big: it makes it so that inductor imports all of aot autograd, which is causing me some cyclical dependency issues when trying to implement AOTAutogradCache (since it needs to be called by aot autograd, but also reuses some inductor stuff). 

Is there any way we can refactor this to its own file instead? Or at least move this import into the function that needs it?